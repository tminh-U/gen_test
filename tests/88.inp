11
19 19 12 20 12 13 8 8 14 7 4 
12
10 10
2 5
8 10
1 11
3 7
7 10
4 4
11 11
3 10
8 9
5 9
3 6
