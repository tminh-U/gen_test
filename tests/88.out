2
8
5
32
11
9
2
2
25
4
14
10
